src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/service/OffenceService.kt

```diff
@@ -85,7 +84,7 @@ class OffenceService(
       description = offence.description,
       statuteCode = statute,
       hoCode = homeOfficeCode,
-      severityRanking = "99", // TODO should derive this?
+      severityRanking = if (offence.category != null) offence.category.toString() else "99",
```

Comment:
I think you could use idiomatic Kotlin here, e.g. `severityRanking = offence.category?.toString ?: "99"`

Reply:
Thanks Mike, that's much neater.. will change in next PR